shared/unattended/JeOS-23.ks

```diff
@@ -0,0 +1,188 @@
+install
+KVM_TEST_MEDIUM
+text
+reboot
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+part / --fstype=ext4 --grow --asprimary --size=1
+
+%packages
+gpgme
```

Comment:
Compare to `JeOS-21` this does not contain `hardlink` and `patch`. Are you 100% sure they are not needed for all the tests we have marked as `JeOS`-friendly? Or is it a dependency in F23?